source/globalCommands.py

```diff
@@ -1,9 +1,9 @@
 # -*- coding: UTF-8 -*-
-#globalCommands.py
-#A part of NonVisual Desktop Access (NVDA)
-#This file is covered by the GNU General Public License.
-#See the file COPYING for more details.
-#Copyright (C) 2006-2018 NV Access Limited, Peter Vágner, Aleksey Sadovoy, Rui Batista, Joseph Lee, Leonard de Ruijter, Derek Riemer, Babbage B.V., Davy Kager, Ethan Holliger, Łukasz Golonka
+# globalCommands.py
+# A part of NonVisual Desktop Access (NVDA)
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+# Copyright (C) 2006-2020 NV Access Limited, Peter Vágner, Aleksey Sadovoy, Rui Batista, Joseph Lee, Leonard de Ruijter, Derek Riemer, Babbage B.V., Davy Kager, Ethan Holliger, Łukasz Golonka, Accessolutions, Julien Cochuyt  # noqa: E501 line too long

```

Comment:
Why not split the line?

Reply:
My intent here was to reduce the number of lines that the header spans across.
Also, if we ever do the proposed automatic update of the copyright holders as extracted from git log, we'll also need to consider what is already present in the header of each file, for two reasons:
 - Some contributors are mentioned in the headers but do not appear in the log as their contribution is older than the repo. Eg. Peter Vágner in `globalCommands.py`
 - Some of us contribute both personally and professionally but most often commit with the same e-mail. Eg. Leonard de Ruijter for Babbage B.V. or myself for Accessolutions

This info will be much easier to extract from each file if it stays on one line.
This being said, there is no strong technical limitations so what do you prefer? Shall I split or let as is?

Reply:
I think for now it's better to split it, we can always programmatically join the lines again if necessary.

Reply:
Done: 25b87fd

I'll try to apply this same guideline to my other pending PRs asap.